From ca92801bcdc702c426977f99b26569452967714d Mon Sep 17 00:00:00 2001
Message-Id: <ca92801bcdc702c426977f99b26569452967714d.1421709247.git.chang-joon.lee@intel.com>
In-Reply-To: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
References: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
From: Gajanan Bhat <gajanan.bhat@intel.com>
Date: Tue, 11 Nov 2014 19:37:12 +0530
Subject: [PATCH 68/75] REVERTME [VPG]: adf/intel: Comment out acquire fence
 logic

Its failing always. need to rework on that. Also set hotplug=1

For: GMINL-3547
Change-Id: Ic48dc916478cb62acec501e56abbcc136f28fab7
Signed-off-by: Gajanan Bhat <gajanan.bhat@intel.com>
Signed-off-by: Uma Shankar <uma.shankar@intel.com>
---
 drivers/video/adf/adf_fops.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/video/adf/adf_fops.c b/drivers/video/adf/adf_fops.c
index 1de8ef1..8660c76 100644
--- a/drivers/video/adf/adf_fops.c
+++ b/drivers/video/adf/adf_fops.c
@@ -196,7 +196,7 @@ static int adf_buffer_import(struct adf_device *dev,
 		buf->pitch[i] = user_buf.pitch[i];
 	}
 	buf->n_planes = user_buf.n_planes;
-
+#if 0
 	if (user_buf.acquire_fence >= 0) {
 		buf->acquire_fence = sync_fence_fdget(user_buf.acquire_fence);
 		if (!buf->acquire_fence) {
@@ -206,7 +206,7 @@ static int adf_buffer_import(struct adf_device *dev,
 			goto done;
 		}
 	}
-
+#endif
 done:
 	if (ret < 0)
 		adf_buffer_cleanup(buf);
@@ -585,7 +585,7 @@ static int adf_intf_get_data(struct adf_interface *intf,
 
 	mutex_lock(&dev->client_lock);
 	read_lock_irqsave(&intf->hotplug_modelist_lock, flags);
-	data.hotplug_detect = intf->hotplug_detect;
+	data.hotplug_detect = 1; /* intf->hotplug_detect; */
 	modelist_size = min(data.n_available_modes, intf->n_modes) *
 			sizeof(intf->modelist[0]);
 	memcpy(modelist, intf->modelist, modelist_size);
-- 
1.7.9.5

